### PR TITLE
Update StatefulSet set docs for autodelete graduation to beta

### PR DESCRIPTION
**Pull request by mattcary**
Placeholder PR.

[KEP 1847](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/1847-autoremove-statefulset-pvcs)

[Implementation PR #99728](https://github.com/kubernetes/kubernetes/pull/99728)

### CODE REVIEW

**Comment by Rishit-dagli** [comment]
Hello @mattcary 👋 
I’m reaching out from the 1.27 Release Docs team. The Docs PRs reviewed and are ready to merge deadline is 4th April 2023 and this PR is still a draft PR, could you please populate the PR to be ready for review as a first step.

Also, the docs PR should be made against the [`dev-1.27` branch](https://github.com/kubernetes/website/tree/dev-1.27).

**Comment by mattcary** [comment]
Oops I made https://github.com/kubernetes/website/pull/40468, sorry for the confusion. I'll transfer those changes into this PR.

**Comment by mattcary** [comment]
@Rishit-dagli okay, this is update to the correct PR and branch. Sorry for my mistake.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -191,7 +191,8 @@ For a reference to old feature gates that are removed, please refer to
 | `ServiceNodePortStaticSubrange` | `false` | Alpha | 1.27 | |
 | `SizeMemoryBackedVolumes` | `false` | Alpha | 1.20 | 1.21 |
 | `SizeMemoryBackedVolumes` | `true` | Beta | 1.22 | |
-| `StatefulSetAutoDeletePVC` | `false` | Alpha | 1.22 | |
+| `StatefulSetAutoDeletePVC` | `false` | Alpha | 1.22 | 1.27 |
+| `StatefulSetAutoDeletePVC` | `false` | Beta | 1.28 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:195]: are you serious? We are documenting something for 1.28 now?

Comment (mattcary) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:195]: No, I just got confused about what release cycle we're on... fixed

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Comment by reylejano** [comment]
Reran the deploy from Netlify and the preview built successfully:
https://deploy-preview-39926--kubernetes-io-vnext-staging.netlify.app/

Preview of updated pages:
- https://deploy-preview-39926--kubernetes-io-vnext-staging.netlify.app/docs/concepts/workloads/controllers/statefulset
- https://deploy-preview-39926--kubernetes-io-vnext-staging.netlify.app/docs/reference/command-line-tools-reference/feature-gates

**Comment by reylejano** [comment]
To re-trigger the checks for this PR to merge, @mattcary can you either `git commit --amend --date=now` or `git commit --allow-empty` and force push to your fork

**Comment by mattcary** [comment]
Done! thanks for tips on negotiating with CI robot.

**Pull request merged**